build.cake

```diff
@@ -242,6 +243,26 @@ foreach (var runtime in new[] { "netcoreapp2.1", "netcoreapp3.1" })
     testNetStandard20.IsDependentOn(task);
 }
 
+var testNetCore21 = Task("TestNetCore21")
```

Comment:
I must be overlooking something, but how is this different from the above section for NetStandard20? The only differences I found diffing was in strings for names and descriptions, but how do we know that the tests are run against the .NET Core 2.1 version of the framework?